R/RequiredCols.R

```diff
@@ -449,10 +452,71 @@ TADA_CheckRequiredFields <- function(.data) {
 #' reducedcols_Data_Nutrients_UT <- TADA_RetainRequired(Data_Nutrients_UT)
 #'
 TADA_RetainRequired <- function(.data) {
```

Comment:
This needs review. I combined functions TADA_RetainRequired and TADA_AutoFilter here. Does that make sense or are they may be better left as separate functions? Possible new name suggestion: maybe TADA_ReduceCols